internal/service/imagebuilder/component.go

```diff
@@ -131,7 +133,7 @@ func ResourceComponent() *schema.Resource {
 
 func resourceComponentCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
-	conn := meta.(*conns.AWSClient).ImageBuilderConn(ctx)
+	conn := meta.(*conns.AWSClient).ImageBuilderClient(ctx)
```

Comment:
I noticed that this variable could now be called `client` instead of `conn` at about 50 places in this PR.
Just meant as a note, feel free to ignore.